.travis.yml

```diff
@@ -14,6 +14,10 @@ before_install:
       if [ "$SKIP_SERVER_INSTALL" != "true" ]; then
         docker-compose -f .travis-docker-compose.yml up -d
       fi
+    - wget http://launchpadlibrarian.net/433926958/libmagic-mgc_5.37-3_amd64.deb
+    - wget http://launchpadlibrarian.net/433926961/libmagic1_5.37-3_amd64.deb
+    - sudo dpkg -i libmagic-mgc_5.37-3_amd64.deb
+    - sudo dpkg -i libmagic1_5.37-3_amd64.deb
```

Comment:
I found an issue with `libmagic<5.37` (both on OSX and bionic), it does not recognize `xlsx` and returns `application/octet-stream` which is critical for Belg CDN. libmagic 5.37/5.38 does the job.
Bionic has a 5.32 (or lower).
I will mention it in the release notes that libmagic>=5.37 is required.